analysis/bulk-deconvolution/exploratory-notebooks/compare-epic-quantiseq.Rmd

```diff
@@ -0,0 +1,441 @@
+  purrr::walk(
+    \(pid) {
+            
+      # determine which cell types to plot
+      if (pid %in% tref_projects) {
+        plot_celltypes <- tref_shared
+        color_map <- tref_shared_color_map
+      } else {
+        plot_celltypes <- bref_shared
+        color_map <- bref_shared_color_map
+      }
+  
+      # Subset to only this project and update celltypes to be:
+      # - In the shared vector
+      # - "Other"
+      # - not other but unshared
+      project_df <- consensus_macrophages_df |>
+        dplyr::filter(project_id == pid) |>
+        tidyr::drop_na() |>
+        dplyr::mutate(
+          celltype_grouped = ifelse(
+            celltype_consensus %in% c("Other", plot_celltypes), 
+            celltype_consensus,
+            other_celltypes_label
+          ),
+          celltype_grouped = forcats::fct_relevel(celltype_grouped, names(color_map)),
+          celltype_grouped = forcats::fct_rev(celltype_grouped)
+        ) 
+        
+      
+      print(make_barplot(project_df, color_map, cell_type_column = celltype_grouped))
+      
+    }
+  )
+```
+
+
+### Conclusions
+
+
+* Across all inferences from both methods, B cells are fairly uncommon, but `quanTIseq` generally estimates higher fractions compared to `EPIC`
+* Considering projects `SCPCP000001` and `SCPCP000002` (high grade and low grade gliomas, respectively):
+	* NK cells are commonly identified by `quanTIseq` but not by `EPIC`
+	* There appears to be some amount of consistency between macrophage inferences between methods
+	* `EPIC` estimates substantial portions of endothelial cells, which are not present in `quanTIseq`. Similarly, `quanTIseq` estimates substantial proportions of dendritic cells, which are not present in `EPIC`.
+* For project `SCPCP000006` (wilms):
+	* CD8 T cells as well as monocytes dominate the `EPIC` results, and while we don't see CD8 as commonly in `quanTIseq`, we do also see monocytes although often in _different_ samples from where this cell type is common according to `EPIC`
+	* `EPIC` results here have fewer cell types in general compared to `quanTIseq`, even considering the difference in references
+* For project `SCPCP000009` (cns tumors):
+	* We see similar patterns to `SCPCP000001` and `SCPCP000002` where NK cells are commonly identified by `quanTIseq` but not by `EPIC`,  `EPIC` estimates substantial portions of endothelial cells, and `quanTIseq` estimates substantial proportions of dendritic cells
+	* We also see T cells, in particular CD4, dominating in `EPIC` but not in `quanTIseq`
+	* In the sample where `EPIC` classifies nearly 100% of the cells, there are similar cell types between methods but vastly different proportions
+* For project `SCPCP000017` (osteo):
+	* Monocytes and CD4 T cells dominate in `EPIC` but these are much less common in `quanTIseq
+	* In the sample where `EPIC` classifies nearly 100% of the cells, the cell types dominating in `EPIC` tend not to be those dominating in `quanTIseq`.
```

Comment:
I think this is a good observation! We don't expect all cells to be normal, so the fact that we see this in EPIC but not quanTIseq might also be an argument for quanTIseq.